src/Microsoft.TemplateEngine.Utils/TemplateInfoExtensions.cs

```diff
@@ -30,7 +30,7 @@ public static class TemplateInfoExtensions
         /// </summary>
         /// <param name="template">template definition.</param>
         /// <returns>The type defined in the template or null if no type is defined.</returns>
-        /// <remarks>The tags are read in <see cref="SimpleConfigModel.ConvertedDeprecatedTagsToParameterSymbols"/> method. The single value for the tag is guaranteed.</remarks>
+        //The tags are read in <see cref="SimpleConfigModel.ConvertedDeprecatedTagsToParameterSymbols"/> method. The single value for the tag is guaranteed.</remarks>
```

Comment:
Is this line helping the users, or is it just for us? Depending on the answer, lets either:
- Add `<remarks>` at the beginning of the line, or
- Remove the `</remarks>` at the end of the line.

Reply:
Same applies to the method above.

Reply:
This comment should be not public, already removed all the tags.